src/Connection.php

```diff
@@ -202,7 +202,7 @@
      */
     public function ping(): void
     {
-        $this->getMongoClient()->getManager()->selectServer(new ReadPreference(ReadPreference::PRIMARY_PREFERRED));
+        $this->getMongoClient()->getManager()->selectServer(new ReadPreference(ReadPreference::PRIMARY_PREFERRED), true);
```

Comment:
We can't dismiss outdated messages like this?

Reply:
Not sure if it was there before or only because you commented, but I see a "Resolve conversation" button here. In the code scanning results (see the link in the first comment) the error is marked as "fixed".